aptos-move/framework/aptos-framework/sources/dispatchable_fungible_asset.move

```diff
@@ -42,12 +42,14 @@ module aptos_framework::dispatchable_fungible_asset {
         withdraw_function: Option<FunctionInfo>,
         deposit_function: Option<FunctionInfo>,
         derived_balance_function: Option<FunctionInfo>,
+        derived_supply_function: Option<FunctionInfo>
```

Comment:
Adding argument to a public function is not allowed in the upgrade process unfortunately. You will need another function for this.